environments/se/core/src/main/java/org/jboss/weld/environment/se/WeldContainer.java

```diff
@@ -123,6 +123,23 @@ public static WeldContainer instance(String id) {
     }
 
     /**
+     * A convenient method for CDI.current(). Returns current WeldContainer instance
+     * if there is exactly one instance running. Throws IllegalStateException in any other case.
+     * @return current WeldContainer instance iff exactly one instance exists
```

Comment:
Well, if it's abbreviation then use upper case please. But I would rather avoid using similar abbreviations in javadoc...